tests/launcher/configs.yaml

```diff
@@ -0,0 +1,9 @@
+################################################################################
+##                             Config                                         ##
+################################################################################
+config:
+    root_path: "tests/launcher/node/"
+    files:
+        - "0/config.yaml"
+        - "1/config.yaml"
+        - "2/config.yaml"
```

Comment:
I'm a bit on the fence about this. I imagined it would just be a matter of doing `./agora-multi file1.yaml file2.yaml file3.yaml` and voila. Do you think there's a need for a specific, extra configuration ?

Reply:
Yes, I'll think about this more.

Reply:
Now I know what you mean. That's a good idea. It will be much easier to use. I'll fix it.